woocommerce/woocommerce-android
# Manage analytic configuration UI

Closes: #10701
Part of #10699

### Why
As part of Expanded Analytics Hub milestone 2, we plan to add support for customizing the cards we display on the Analytic Hub screen.

### Description
This PR adds the basic UI and Navigation for the Manage Analytics screen. The code added here still needs to handle logic for saving or getting the existing analytic cards.

### Testing instructions
TC1
1. Open the app
2. Tap on the See more button to navigate to the Analytic Hub screen
3. Check that on the top right corner of the screen, a pencil icon is displayed
4. Tap on the pencil icon and check that the app navigated to the Manage Analytics screen
5. Tap on the card items and check that the card selection changes
6. Tap on the close button (x) at the top of the screen and check that a dialog asking to discard changes is displayed.
7. Tap on Keep editing and check that the dialog dismisses 
8. Tap on the close button again and when the dialog appears tap on the Discard button and check that the app now closes the Manage Analytics screen.
9. Enter the Manage Analytics screen again by tapping on the pencil icon
10. Press the save button and check that the app now closes the Manage Analytics screen.

### Images/gif
| Pencil  icon | Manage analytics screen | Discard changes dialog |
| ------------- | ------------- | ------------- |
| <img width="300" src="https://github.com/woocommerce/woocommerce-android/assets/18119390/e8aac21b-261f-46fc-bb6b-162be3200724" /> | <img width="300" src="https://github.com/woocommerce/woocommerce-android/assets/18119390/19165475-d004-47a2-806c-665ff7dcc6d2" />  | <img width="300" src="https://github.com/woocommerce/woocommerce-android/assets/18119390/9d783e0d-31b3-47fd-a769-93db4b99c955" /> |
